README.md

```diff
@@ -43,43 +53,43 @@ End of old workshop links -->
 
 ## Contacts
 
-This effort is being coordinated by [Zeljko Ivezic](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/new?body=@ivezic) and [Beth Willman](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/new?body=@bethwillman), while [Phil Marshall](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/new?body=@drphilmarshall) is the white paper's editor-in-chief. Any of them can propagate your privately-communicated concerns into a redacted issue on this repository. Contributions are very welcome from all round the LSST science collaborations, and beyond. Perhaps we are missing a science case? Or an idea for how to perturb the observing strategy? We'd like to hear from you! Please send all your feedback to [this repo's issues](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues).
+This effort is being coordinated by [Phil Marshall](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/new?body=@drphilmarshall) (the v1 COSEP's editor-in-chief), with guidance from [Zeljko Ivezic](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/new?body=@ivezic) and [Lynne Jones](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/new?body=@rhiannonlynne). Any of them can propagate your privately-communicated concerns into a redacted issue on this repository. Contributions are very welcome from all round the LSST science collaborations, and beyond. Perhaps we are missing a science case? Or an idea for how to perturb the observing strategy? We'd like to hear from you! Please send all your feedback to [this repo's issues](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues).
 
-All white paper content is Copyright 2015, 2016, 2017 The Authors. If you make use of the ideas and results in the white paper in your research, please cite it as **"(LSST Science Collaborations 2017)"**, and provide the URL of this repository: https://github.com/LSSTScienceCollaborations/ObservingStrategy. The BibTeX entry for the white paper is below.  Thanks!
+All COSEP content is Copyright 2015, 2016, 2017, 2018 The Authors. If you make use of the ideas and results in the COSEP in your research, please cite it as **"(LSST Science Collaborations 2017)"**, and provide the URL of this repository: https://github.com/LSSTScienceCollaborations/ObservingStrategy. The BibTeX entry for the v1 COSEP is below.  Thanks!
```

Comment:
This says that COSEP content is copyright 2018 but the reference is 2017. Looks like maybe a mismatch

Reply:
Yes - I think we can just say "Copyright 2017" and leave it at that. Thanks!